src/components/overrides/Header.astro

```diff
@@ -55,7 +55,10 @@ const props = {
   </div>
   <div class="sl-flex flex-row items-center gap-4">
     <div class="sl-flex items-center">
-      <a href="/blog" class="no-underline text-accent-600 dark:text-accent-200"> Blog </a>
+      <a href="/blog" class="no-underline text-accent-600 dark:text-accent-200">Blog</a>
+    </div>
+    <div class="sl-flex items-center">
+      <a href="/docs" class="no-underline text-accent-600 dark:text-accent-200">Documentation</a>
```

Comment:
Just an idea. The "documentation" word is lengthy and will occupy header on mobile. Maybe we can use [a Starlight icon](https://starlight.astro.build/reference/icons/#all-icons)? `document` or `open-book`, wdyt?